themes/frontierline-firefox/style.css

```diff
@@ -52,52 +41,22 @@ Version:        1.0
 }
 
 #newsletter-subscribe.newsletter-firefox .form-title h3 {
+    font-size: 1.1666rem;
```

Comment:
rem needs to be below px or it doesn't take affect.

Reply:
Doh!